topics/epigenetics/tutorials/cut_and_run/tutorial.md

```diff
@@ -198,8 +198,8 @@ The FastQC report pointed out that we have in our data some standard Illumina ad
 >
 > > <solution-title></solution-title>
 > >
-> > 1. ~55% and ~57%
-> > 2. 100%
+> > 1. ~55% for Read 1 and ~57% for Read 2
+> > 2. The last line indicates that 3.5% of pairs have been removed.
```

Comment:
@heylf could you confirm it makes sense (the output of bowtie2 does not have 300k reads as input).

Reply:
Yeah thats fine